dbt/models/reporting/reporting.ratio_stats.py

```diff
@@ -249,7 +141,13 @@ def ccao_prb(fmv, sale_price):
 
 
 def report_summarise(df, geography_id, geography_type):
-    # Aggregates data and calculates summary statistics for given groupings
+    """
+    Aggregates data and calculates summary statistics for given groupings
+    """
+
+    # Convert the Spark input dataframes to Pandas for compatibility
+    # with assesspy functions
+    df = df.toPandas()
```

Comment:
This line is absolutely _critical_ to making this work. It converts the spark table returned by `dbt-athena` (in this case all of `reporting.ratio_stats_input`) to a pandas data frame. IMO, we should refactor this to use pyspark since it will probably be much faster.